pkg/harvester-manager/list/harvesterhci.io.management.cluster.vue

```diff
@@ -212,7 +213,11 @@ export default {
   methods: {
     async getHarvesterRepository() {
       try {
-        return await getHelmRepository(this.$store, HARVESTER_REPO.spec.gitRepo, HARVESTER_REPO.spec.gitBranch);
+        if (isRancherPrime()) {
+          return await getHelmRepositoryExact(this.$store, HARVESTER_REPO.gitRepo);
+        } else {
+          return await getHelmRepositoryMatch(this.$store, communityRepoRegexes);
```

Comment:
Non-Prime users can manually add Harvester repository. The Harvester repo should match one of below:

Git repo url:
- https://github.com/harvester/harvester-ui-extension
- https://github.com/harvester/harvester-ui-extension/
- https://github.com/{forker-repository}/harvester-ui-extension
- https://github.com/{forker-repository}/harvester-ui-extension/
GitHub deployments url:
- https://harvester.github.io/harvester-ui-extension